packages/generator/src/GenerateContext/ForeignType.ts

```diff
@@ -20,28 +20,25 @@ import { AbstractImportable } from "./EnhancedBase.js";
 export class ForeignType extends AbstractImportable {
   constructor(
     public _common: EnhanceCommon,
-    public readonly type: string,
-    apiNamespace: string,
+    apiNamespace: string | undefined,
     shortApiName: string,
+    destinationPackage: string,
   ) {
     super(
       _common,
-      `${apiNamespace}.${shortApiName}`,
-      _common.apiNamespacePackageMap.get(apiNamespace)!,
+      apiNamespace ? `${apiNamespace}.${shortApiName}` : shortApiName,
```

Comment:
This is because of the limitation in dev console right now

Reply:
That is correct. In an ideal world you could just assume foreign types have an api namespace